images/linux/scripts/installers/go.sh

```diff
@@ -13,16 +13,16 @@ golangTags="/tmp/golang_tags.json"
 #   $2=IsDefaultVersion (true or false)
 function InstallGo () {
     version=$( getFullGoVersion $1 )
-    downloadVersion=$version.linux-amd64.tar.gz
-    goFolder=/usr/local/go$1
```

Comment:
should we setup a symlink for now so that it's not such a breaking change?

Reply:
Honestly, I don't think that symlinks are required here because we have never announced these paths anywhere.
But probably symlinks won't be worse

Reply:
Added symlinks.